### ISSUE TITLE
Cluster configuration structure for launching and provisioning 

### ISSUE THREAD

**Issue by cjh1** [Kitware/cumulus#204]
```javascript
{
    "type": "ec2",
    "profileId": "profile id",
    "config": {
        "launch": {
            "spec": "spec name aka playbook",
            "params": {
                ...
            }
        },
        "provision": {
            "spec": "spec name aka playbook",
            "params": {
                ...
            },
            "ssh": {
                "user": "user to use for provisioning"
            }
        }
    }
}

```

**Comment by cjh1** [comment]
This documents the solution to #184 and #183.

**Comment by cjh1** [comment]
Resolved by #207
